content/swift/concepts/protocols/terms/encodable/encodable.md

```diff
@@ -12,9 +12,15 @@ CatalogContent:
   - 'paths/build-ios-apps-with-swiftui'
 ---
 
-**`Encodable`** is a protocol that enables a type's data to be encoded for use with Application Programming Interfaces ([APIs](https://www.codecademy.com/resources/docs/general/api)).
+**`Encodable`** is a protocol that enables a type's data to be encoded and for use with Application Programming Interfaces ([APIs](https://www.codecademy.com/resources/docs/general/api)). If data also needs to be decoded, it's convenient to use the `Codable` protocol because it includes both the `Encodable` and `Decodable` protocols.
 
-> **Note:** If data also needs to be decoded, it's convenient to use the `Codable` protocol because it includes both the `Encodable` and `Decodable` protocols.
+The protocol is related to the following protocols:
+
+- Codeable: 
+- [Decodeable](https://www.codecademy.com/resources/docs/swift/protocols/decodable):
+- CodingKey:
+- CodingKeyRepresentable:
```

Comment:
@BBrandl948 Can you include a brief sentence for each about how it is related

Reply:
I will try to

Reply:
I hope my suggestions meet your intentions. Committed all changes